README.md

```diff
@@ -8,110 +8,88 @@
 ![CI Check Golang](https://github.com/intel/confidential-cloud-native-primitives/actions/workflows/pr-golang-check.yaml/badge.svg)
 ![CI Check Container](https://github.com/intel/confidential-cloud-native-primitives/actions/workflows/pr-container-check.yaml/badge.svg)
 ![CC Foundation Image Customize](https://github.com/intel/confidential-cloud-native-primitives/actions/workflows/image-rewriter.yaml/badge.svg)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/8325/badge)](https://www.bestpractices.dev/projects/8325)
 
 ## 1. Introduction
 
-Confidential Computing technology like Intel TDX provides isolated encryption runtime
+Confidential Computing technologies like Intel® TDX provides an isolated encryption runtime
 environment to protect data-in-use based on hardware Trusted Execution Environment (TEE).
 It requires a full chain integrity measurement on the launch-time or runtime environment
-to guarantee "consistently behavior in expected way" (defined by
-[Trusted Computing](https://en.wikipedia.org/wiki/Trusted_Computing)) of confidential
+to guarantee "consistent behavior in an expected way" of confidential
 computing environment for tenant's zero-trust use case.
 
 This project is designed to provide cloud native measurement for the full measurement
 chain from TEE TCB -> Firmware TCB -> Guest OS TCB -> Cloud Native TCB as follows:
 
 ![](/docs/cc-full-meaurement-chain.png)
 
-_NOTE: Different with traditional trusted computing on non-confidential environment,
+_NOTE: Different from traditional trusted computing on non-confidential environment,
 the measurement chain is not only started with Guest's `SRTM` (Static Root Of Measurement)
-but also need include the TEE TCB, because the CC VM environment is created by TEE
-via `DRTM` (Dynamic Root of Measurement) like Intel TXT on the host._
+but it also needs to include the TEE TCB, because the CC VM environment is created by TEE
```

Comment:
This is a small grammar correction I missed before   -->  "(Static Root of Mesurement), but it also needs to include the TEE TCB because the"  The comma should be after the parenthesis and before b"ut" because it is a compound sentence. The "because" is a subordinating conjunction (had to look that terminology up) so no comma is needed.  English can be very nuanced.

Reply:
Thanks for the English learning tips :-)

Reply:
I have to look these things up myself to re-remember :)